txnkv/transaction/2pc.go

```diff
@@ -2049,16 +2049,18 @@ func (batchExe *batchExecutor) process(batches []batchMutations) error {
 		return err
 	}
 
-	// For prewrite, stop sending other requests after receiving first error.
+	// For prewrite and flush, stop sending other requests after receiving first error.
```

Comment:
If there is such case for `prewrite`, we could add one for `flush` too.

Reply:
Yes. I've checked all type assertions. Seems the only thing left that make them "unequal" is stats.